src/style.css

```diff
@@ -0,0 +1,107 @@
+:root {
+  --font-color: #3d4e61;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 600;
+}
+
+.search {
+  display: flex;
+  height: 70px;
+  justify-content: center;
+  margin-top: 20px;
+  width: 100%;
+}
+
+.search--small {
+  height: 42px;
+}
+
+.search__label {
+  position: relative;
+  width: 100%;
+}
+
+.search__label::after {
+  position: absolute;
+  content: "";
+  display: block;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.search__label--big::after {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search__label--small::after {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search__input {
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: var(--font-color);
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  outline: none;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search__input--big {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search__input--small {
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
```

Comment:
2'nd point in checklist, create one base class and second one to only change certain styles